algorithms/rainbow/main.py

```diff
@@ -0,0 +1,178 @@
+"""
+Adapted from https://github.com/Kaixhin/Rainbow
+
+The main file needed within rainbow. Runs of the train and test functions from their respective
+files.
+
+Example of use:
+`cd algorithms/rainbow`
+`python main.py`
+
+Runs Rainbow DQN on our AI2ThorEnv wrapper with default params. Optionally it can be run on any
+atari environment as well using the "game" flag, e.g. --game space_invaders.
+"""
+import argparse
+from datetime import datetime
+import torch
+import numpy as np
+
+from algorithms.rainbow.agent import Agent
+from algorithms.rainbow.env import Env, MultipleStepsEnv
+from algorithms.rainbow.memory import ReplayMemory
+from algorithms.rainbow.test import test
+
+from gym_ai2thor.envs.ai2thor_env import AI2ThorEnv
+
+parser = argparse.ArgumentParser(description='Rainbow')
+parser.add_argument('--seed', type=int, default=123, help='Random seed')
+parser.add_argument('--disable-cuda', action='store_true', help='Disable CUDA')
+parser.add_argument('--game', type=str, default='ai2thor', help='ATARI game or environment')
+parser.add_argument('--T-max', type=int, default=int(50e6), metavar='STEPS',
+                    help='Number of training steps (4x number of frames)')
+parser.add_argument('--max-episode-length', type=int, default=int(1e3), metavar='LENGTH',
+                    help='Max episode length (0 to disable)')
+parser.add_argument('--history-length', type=int, default=1, metavar='T',
+                    help='Number of consecutive states processed')
+parser.add_argument('--hidden-size', type=int, default=512, metavar='SIZE',
+                    help='Network hidden size')
+parser.add_argument('--noisy-std', type=float, default=0.8, metavar='σ',
+                    help='Initial standard deviation of noisy linear layers')
+parser.add_argument('--atoms', type=int, default=51, metavar='C',
```

Comment:
maybe num_atoms would be better?

Reply:
having to write num-atoms is 4 more chars on the terminal/cmd and for me it seems to clarify nothing, since you don't have more than 1 param related to atoms. But if you still want to go with this confirm

Reply:
no one actually writes any parameters in haha. No one will ever change it from 51 and if they do it'll be the same length as `--noisy-std` above. 0 problems. 
People might actually get confused that these aren't the atoms, it's just the number of atoms